spec.html

```diff
@@ -14455,7 +14456,7 @@ <h1>Runtime Semantics: Evaluation</h1>
           1. If _importMeta_ is ~empty~, then
             1. Set _importMeta_ to ! OrdinaryObjectCreate(*null*).
             1. Let _importMetaValues_ be ! HostGetImportMetaProperties(_module_).
-            1. For each Record { [[Key]], [[Value]] } _p_ that is an element of _importMetaValues_, do
+            1. For each Record { [[Key]], [[Value]] } _p_ of _importMetaValues_, do
```

Comment:
```suggestion
            1. For each Record _p_ of _importMetaValues_, do
```

I don't think mentioning the fields adds anything here.

Reply:
Since these are anonymous Records, the fields are essentially part of the type. I'd lightly prefer to keep them.

Reply:
could we make them proper named records instead?

Reply:
We could, but this and [the host-defined AO which provides this list](https://tc39.es/ecma262/#sec-hostgetimportmetaproperties) are the only two places which talk about this particular kind of record, so I think adding another layer of indirection there would hurt rather than help clarity.